src/commands/Reactions2/Baka.ts

```diff
@@ -0,0 +1,51 @@
+import { MessageType, Mimetype } from '@adiwajshing/baileys'
+import MessageHandler from '../../Handlers/MessageHandler'
+import BaseCommand from '../../lib/BaseCommand'
+import WAClient from '../../lib/WAClient'
+import { ISimplifiedMessage } from '../../typings'
+import { tmpdir } from 'os'
+import { exec } from 'child_process'
+import { readFile, unlink, writeFile } from 'fs/promises'
+import { promisify } from 'util'
+
+export default class Command extends BaseCommand {
+    constructor(client: WAClient, handler: MessageHandler) {
+        super(client, handler, {
+            command: 'baka',
+            description: 'said baka to someone',
+            category: 'reactions2',
+            usage: `${client.config.prefix}baka [tag/quote users]`
+        })
+    }
+    exec = promisify(exec)
+
+    GIFBufferToVideoBuffer = async (image: Buffer): Promise<Buffer> => {
+        const filename = `${tmpdir()}/${Math.random().toString(36)}`
+        await writeFile(`${filename}.gif`, image)
+        await this.exec(
+            `ffmpeg -f gif -i ${filename}.gif -movflags faststart -pix_fmt yuv420p -vf "scale=trunc(iw/2)*2:trunc(ih/2)*2" ${filename}.mp4`
+        )
+        const buffer = await readFile(`${filename}.mp4`)
+        Promise.all([unlink(`${filename}.mp4`), unlink(`${filename}.gif`)])
+        return buffer
+    }
+    run = async (M: ISimplifiedMessage): Promise<void> => {
+        if (M.quoted?.sender) M.mentioned.push(M.quoted.sender)
+        if (!M.mentioned.length) M.mentioned.push(M.sender.jid)
+        M.reply(
+            await this.GIFBufferToVideoBuffer(
```

Comment:
Instead of declaring the GIFBufferToVideoBuffer inside the command. Can you use the one present in utils?
Like https://github.com/PrajjwalDatir/Kaoi/blob/main/src/commands/Fun/Ship.ts#L57